README.md

```diff
@@ -2,73 +2,66 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/aerogear/mobile-cli/badge.svg?branch=add-go-report-card-to-readme)](https://coveralls.io/github/aerogear/mobile-cli?branch=add-go-report-card-to-readme)
 
+# Mobile CLI
 
-## The Mobile CLI is a standalone CLI that can also be used a kubectl / oc plugin
+**NOTE: The Mobile CLI is still under construction and not yet fit for use.**
 
-## Note this is still under construction and not yet fit for use
+The Mobile CLI is a standalone CLI that can also be used as a kubectl or oc plugin.
 
-The mobile CLI focuses on a small set of commands to empower mobile focused developers to consume and take full advantage of the RedHat mobile suite
-of services ontop of Kubernetes / OpenShift. 
+It focuses on a small set of commands to empower mobile focused developers to consume and take full advantage of the RedHat mobile suite of services ontop of Kubernetes/OpenShift. 
 
-It uses a language familiar to mobile developers and abstracts away some of the complexity of dealing with Kubernetes / OpenShift which can be 
-initially daunting and overwhelming.
+It uses a language familiar to mobile developers and abstracts away some of the complexity of dealing with Kubernetes/OpenShift which can be initially daunting and overwhelming.
 
-### Examples
-Note not all of these commands currently exist but are present below to show the general concept
+## Examples
+**Note: Not all of these commands currently exist but are present below to show the general concept.**
 
-```
+```bash
 mobile get services
-
-mobile provision fh-sync
-
-mobile --namespace=myproject get clientconfig <MobileClientID>
-
-mobile create integration fh-sync keycloak
-
-mobile create clientbuild <MobileClientID> <Git_Source_Url> [buildName]
-
-mobile get buildartifact <clientBuildID> 
+mobile create serviceinstance fh-sync-server --namespace=myproject
+mobile get clientconfig <MobileClientID> --namespace=myproject 
+mobile create integration <consuming_service_instance_id> <providing_service_instance_id> --namespace=myproject
+mobile create clientbuild <MobileClientID> <Git_Source_Url> <buildName>
+mobile get clientbuilds <clientBuildID> 
 ``` 
 
-### Checkout 
+## CLI Installation
+### Pre-requisites
+- Have a local Kubernetes or OpenShift cluster with mobile clients and services available via minikube, [mobile core installer](https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/local-setup.adoc) or [minishift](https://github.com/aerogear/minishift-mobilecore-addon).
+- Install [glide](https://github.com/Masterminds/glide) (e.g. `brew install glide [Mac]`)
```

Comment:
Can we get installation docs added for at least RHEL here too?

Reply:
@philbrookes  I think this can be removed since this information should be within the linked guide. wdyt?

Reply:
yep, that's even better.